css/modules/aioseop_module.css

```diff
@@ -1264,6 +1227,57 @@ div.aioseop_notice a.aioseop_dismiss_link {
 	vertical-align: bottom;
 }
 
+.aioseop_tabs.ui-widget-content {
+	background: none !important;
+}
+
+.aioseop_tab.ui-widget-content {
+	border: 1px solid #aaa !important;
+}
+
+.aioseop_tab.ui-widget-content a.aioseop_help_text_link {
+	color: #888 !important;
+}
+
+.aioseop_tabs.ui-widget {
+	font-size: 13px !important;
+	border: none !important;
+}
+
+.aioseop_tabs .ui-widget-header {
+	border: none !important;
+	background: none !important;
+	border-bottom-right-radius: 0px !important;
+	border-bottom-left-radius: 0px !important;
+}
+
+.aioseop_tabs ul {
+	margin-left: .2em !important;
+}
+
+.aioseop_header_tabs li a.aioseop_header_tab {
+	border-bottom: none !important;
+}
+
+.aioseop_tabs .ui-tabs .ui-tabs-panel {
+	border-width: inherit !important;
+}
+
+.aioseop_tabs .ui-state-default,
+.aioseop_tabs .ui-widget-content .ui-state-default,
+.aioseop_tabs .ui-widget-header .ui-state-default {
+	background-image: none !important;
+}
+
+.aioseop_tabs .ui-state-active a {
+	font-weight: bold !important;
+}
+
+.aioseop_tabs.ui-tabs .ui-tabs-nav li {
```

Comment:
@arnaudbroes as discussed with @wpsmort it seems you were seeing something different from him and thats why you suggested the last point. I misunderstood it (that's why in the future please attach screenshots so that we see the same thing) and removed the bottom border from all tabs. Steve was trying to contact you to see if you can reproduce and fix this at your end because what he sees is what I see, which is probably different from what you see.